mev_inspect/aave_liquidations.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+
+from mev_inspect.schemas.classified_traces import (
+    Classification,
+    ClassifiedTrace,
+    # CallTrace,
+    DecodedCallTrace,
+)
+from mev_inspect.schemas.liquidations import Liquidation
+
+
+# Inspect list of classified traces and identify liquidation
+def get_liquidations(traces: List[ClassifiedTrace]):
+    tx = []
+    liquidations = []
+    result = []
+
+    # Protocol contract address must be in included, below is AaveLendingPoolCoreV1
+    addrs = [
```

Comment:
if these are constant, throw them up top in a constant

also no short names

Reply:
done